nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/installation.md

```diff
@@ -47,3 +53,81 @@ To stop nextflow from looking for updates online, you can tell it to run in offl
 ```bash
 export NXF_OFFLINE='TRUE'
 ```
+<<<<<<< HEAD
```

Comment:
This is a merge conflict marker which has been left in when resolving a merge conflict - you can delete it :)